src/test/unit_tests/jest_tests/test_list_object.test.js

```diff
@@ -1,175 +0,0 @@
-/* Copyright (C) 2016 NooBaa */
-'use strict';
-
-
-const fs = require('fs');
-const path = require('path');
-const fs_utils = require('../../../util/fs_utils');
-const nb_native = require('../../../util/nb_native');
-const {TMP_PATH} = require('../../system_tests/test_utils');
-const { get_process_fs_context } = require('../../../util/native_fs_utils');
-
-const tmp_fs_path = path.join(TMP_PATH, 'test_list_object');
-const DEFAULT_FS_CONFIG = get_process_fs_context();
-
-// eslint-disable-next-line max-lines-per-function
-describe('manage list objct flow', () => {
-    describe('Telldir and Seekdir implementation', () => {
```

Comment:
Did you remove this test? or just moved it elsewhere? It just seems like a good test to have..

Reply:
Its not removed it moved inside same file.